src/hotspot/share/gc/shenandoah/shenandoahMarkCompact.cpp

```diff
@@ -114,14 +116,14 @@ void ShenandoahMarkCompact::do_it(GCCause::Cause gc_cause) {
 
     // b. Cancel concurrent mark, if in progress
     if (heap->is_concurrent_mark_in_progress()) {
-      heap->concurrent_mark()->cancel();
+      ShenandoahConcurrentMark::cancel();
```

Comment:
While cancel() doesn't exactly need to be an instance-method, it feels strange to do a static call here...